test/common/README.md

```diff
@@ -4,9 +4,21 @@ This directory contains modules used to test the Node.js implementation.
 
 ## Table of Contents
 
+* [Benchmark module](#benchmark-module)
 * [Common module API](#common-module-api)
 * [WPT module](#wpt-module)
 
+## Benchmark Module
+
+The `benchmark` module is used by tests to run benchmarks.
+
+### runBenchmark(name, args, env)
+
+* `name` [&lt;String>] Name of benchmark suite to be run.
+* `args` [&lt;Array>] Array of environment variable key/value pairs (ex:
```

Comment:
Hmm, I think a better name for the parameter may be "settings"? Also the environment variable seems to be a misnomer, as does the "key/value pair" verbiage which sounds like an array (though the example definitely helps for that case). The "Array" type annotation could be changed to string[] to better represent the member types.

Reply:
Hmm, I was thinking "args" because this just does `--set` for whatever is passed in (args as in command line args). This argument is semi confusing, but I am planning on submitting a PR after this is merged to convert this argument to be an Object so that way it's easier to understand -- using the array for right now is just to avoid a giant diff :)

Reply:
FWIW it's called `configs` in `createBenchmark`, though in a context like this it should be `config`.

> * `configs` {Object} The benchmark parameters. `createBenchmark` will run all
  possible combinations of these parameters, unless specified otherwise.
  Each configuration is a property with an array of possible values.
  Note that the configuration values can only be strings or numbers.